codelabs/raspberry-pi-proximity-sensor/codelab.json

```diff
@@ -0,0 +1,27 @@
+{
+  "environment": "web",
+  "format": "html",
+  "prefix": "https://storage.googleapis.com",
+  "mainga": "UA-49880327-14",
```

Comment:
When you re-export please add the `-ga` option and this will be changed to the Solace GA ID. 
`claat export -ga UA-3921398-10`
(This has been updated in the codelab-4-codelab tutorial as well)

Reply:
Done